### DISCUSSION TITLE
CZK not accepted as foreign currency code?

### DISCUSSION THREAD

**Discussion by Jasper-Dev** [firefly-iii/firefly-iii#6434] [Firefly III] [answered]
Last summer holiday I went to visit the Czech Republic (such a beautiful country), but if I want to import the transactions I made last summer it gives me this error:

` transactions.0.foreign_currency_code: The selected transactions.0.foreign_currency_code is invalid. (original value: "CZK")`

things I did so far:
- Checked if there was already a question or issue about this. (couldn't find one)
- Checked to see if CZK was the correct ISO 4217 code (which according to https://www.iban.com/currency-codes it is).
- Tried to map the field, but couldn't find 'CZK' in the list of 'foreign_currency_code'. (I did find things like: 'Ukrainian hryvnia (UAH)', 'Hungarian forint (HUF)', and 'Polish złoty (PLN)' but no 'Czech Koruna (CZK)')

Can someone help me with this?

**Comment by JC5** [comment, marked as answer]
You have to add it as a currency first.

**Reply by Jasper-Dev** [reply]
Ah I didn't know this was an option, my bad 😅

Out of curiousity though, why isn't the full list of ISO 4217 codes added by default?

**Reply by JC5** [reply]
For the one user who happens to need the Somali shilling, there are thousands of users who have to scroll past it to activate the US dollar.

**Reply by Jasper-Dev** [reply]
Fair enough! 😂
